GcloudBackup.ps1

```diff
@@ -216,7 +221,10 @@ function removeOldBackups() {
 		
 			foreach ($file in $files) {
 				# We remove trim spaces, then replace multiple spaces with one space only and then we split it into variables
-				$fileSize,$fileDate,$filePath = (("$file".trim()) -Replace '\s+', ' ') -Split ' '								
+				$fileSize,$fileDate = (("$file".trim()) -Replace '\s+', ' ').Split(' ')[0,1]
```

Comment:
Here we dodge a bullet, because the folder name or file name might have more than one spacing (by mistake) and would break the program